core/sql/parser/StmtDDLAlterTableAlterColumn.cpp

```diff
@@ -55,16 +55,22 @@ StmtDDLAlterTableAlterColumn::getText() const
 //----------------------------------------------------------------------------
 // CLASS StmtDDLAlterTableAlterColumnDatatype
 //----------------------------------------------------------------------------
-StmtDDLAlterTableAlterColumnDatatype::StmtDDLAlterTableAlterColumnDatatype( 
-     const NAString &columnName 
-     , NAType * natype
-     , CollHeap *heap)
+StmtDDLAlterTableAlterColumnDatatype::StmtDDLAlterTableAlterColumnDatatype(
+     ElemDDLNode * pColumnToAlter
+     ,CollHeap    * heap)
      : StmtDDLAlterTableAlterColumn(DDL_ALTER_TABLE_ALTER_COLUMN_DATATYPE,
-                                    columnName,
+                                    NAString(""),
```

Comment:
yes, you are right, it is not needed for this particular case. But we would need it for other column based operations like renaming a column (there is another jira for rename). So we can keep it around and use it for rename implementation.